app.py

```diff
@@ -260,13 +260,20 @@ def render_message(self, message):
                 return None
         elif subtype == 'file_comment':
             user = self.store.find_user_by_id(message['comment']['user'])
+            if user is None:
+                return None
```

Comment:
In this situation, it will skip rendering the message when it can't find the user in the user list. The error of a missing user seems to be due to Slack API truncating the list of users when fetching. However, for now I think it is a valid patch. Could you put a comment over these lines to inform that this is a temporary hack while we still don't paginate the users?

Reply:
Actually  in this case it's not because of truncated users. It's because the user was removed. The message that caused the issue isn't displayed in the normal Slack client, so I assumed that returning None was a valid case here